sortinghat/core/api.py

```diff
@@ -888,3 +888,104 @@ def _delete_unique_identities(trxl, uids):
     trxl.close()
 
     return to_uid
+
+
+@django.db.transaction.atomic
+def unmerge_identities(ctx, uuids):
+    """
+    Unmerge one or more identities from their corresponding unique identities.
+
+    Use this function to separate a list of `uuid` identities, creating a unique
+    identity for each one. `name` and `email` fields from every identity will be
+    set to the corresponding profile fields when the new unique identity is created.
+    Nor the enrollments or the profile from any parent unique identity of the input
+    identities are modified.
+
+    When a given identity `uuid` is equal to the `uuid` from its parent unique
+    identity, there will be no effect.
+
+    The function raises a `NotFoundError` exception when either any `uuid` from
+    the list does not exist in the registry.
+
+    The function raises an `InvalidValueError` exception when either any `uuid`
+    from the list is `None` or an empty string.
+
+    :param ctx: context from where this method is called
+    :param uuids: List of identifiers of the identities set to be unmerged
+
+    :returns: a list of UniqueIdentity objects as the result of the unmerged identities
+
+    :raises NotFoundError: raised when when either any `uuid` from the list
+        does not exist in the registry
+    :raises InvalidValueError: raised when when either any `uuid` from the
+        list is `None` or an empty string
+    """
+
+    def _find_identities(uuids):
+        """Find the identities to be unmerged from their parent unique identities"""
+
+        identities = []
+        for uuid in uuids:
+            # Check whether input values are valid
+            if uuid is None:
+                raise InvalidValueError(msg="'uuid' cannot be None")
+            if uuid == '':
+                raise InvalidValueError(msg="'uuid' cannot be an empty string")
+
+            try:
+                uid = find_identity(uuid)
+                identities.append(uid)
+            except NotFoundError as exc:
+                raise exc
```

Comment:
If we are not handling the exception, it's better to delete the `try...raise` block.

Reply:
Done.